pylint/testutils/_primer/primer_prepare_command.py

```diff
@@ -31,12 +34,14 @@ def run(self) -> None:
                 commit_string += remote_sha1_commit + "_"
         elif self.config.read_commit_string:
             with open(
-                self.primer_directory / "commit_string.txt", encoding="utf-8"
+                self.primer_directory / f"commit_string_{version_string}.txt",
```

Comment:
f"{commit_string}_{version_string}.txt"

Reply:
Oh that is confusing. Interpolating `commit_string` is actually not desired here, the CI job expects to find an artifact called `commit_string...`. That could have been clarified with a comment.

Reply:
Well perhaps it is ok; I commented this when I merged main to my branch & saw "cp: cannot stat 'commit_string_3.10.txt': No such file or directory". 
But I understand once the first run is complete then it should work.

Reply:
Yeah, if you check the output of the step "last run on main" it points to a run from yesterday, so that's why the file isn't there.